### PR TITLE
Update legal.md

### PR DESCRIPTION
**Pull request by theo-armour**
from: contributor agreement 
To: contributor license agreement (CLA)

Otherwise it's unclear what the 'CLA' in the node.js quote stands for

### CODE REVIEW

**Comment by mlinksva** [comment]
Thanks for pointing this out @theo-armour. I don't want to force use of CLA throughout as the term isn't always used or appropriate, but it does seem worth pointing on the first use. I made an additional commit, not particularly opinionated if the style is horrible.

File: _articles/legal.md [hunk 1]
```diff
@@ -90,7 +90,7 @@ For example, as your project grows it adds dependencies or users, or your compan
 
 Alternatively, you can have contributors agree in advance (via an additional contributor agreement -- see below) to certain license changes under certain conditions, beyond those allowed by your existing open source license. This shifts the complexity of changing licenses a bit. You'll need more help from your lawyers up front, and you'll still want to clearly communicate with your project's stakeholders when executing a license change.
 
-## Does my project need an additional contributor agreement?
+## Does my project need an additional contributor [license] agreement (CLA)?
```
Comment (bkeepers) [inline _articles/legal.md]: The titles are already really long on most of these guides. Could we keep this title the same, and call this out in the paragraph text?

Comment (theo-armour) [inline _articles/legal.md]: Putting the abbreviation in the text is fine with me

**Comment by mlinksva** [comment]
Ok, this is improved, thanks for the suggestion @theo-armour. Further wordsmithing may be warranted, but I'm going to merge now as I'll probably touch this again soon for #301.

**Pull request merged**